### ISSUE TITLE
if the project will be built for Windows？

### ISSUE THREAD

**Issue by mmdlearn** [benoitsteiner/tensorflow-opencl#71]

**Comment by rodburns** [comment]
We would like to add support for ComputeCpp on Windows, but currently we don't have firm timescales. I assume this is what you are asking.

**Comment by NakramR** [comment]
I'm not sure what mmdlearn was asking, but what I was wondering was whether this would allow me to get tf to work with an AMD GPU on Windows.
Should it work as is, is there work needed, is it out of scope?
And in the negative, what kind of work would be needed?

**Comment by rodburns** [comment]
Yes we do plan to make a Windows version of ComputeCpp available that should enable you to work with AMD GPU on Windows, but as I say I am not sure when this will happen right now. We haven't investigated anything around drivers for these GPUs on Windows either so there are still many unknowns.

**Comment by sprtkd** [comment]
When can we expect running tensorflow on amd gpus on windows? Are low end gpus like amd m335 supported?

**Comment by JimSEOW** [comment]
With all the hype around AMD GPU, with the trends that Deep Learning is not just restricted to server, having Windows support is crucial.

**Comment by kyjak** [comment]
So  the blocker is only the computecpp library? This should be really the top priority for this project, the windows availability.
